pass/ProteusPass.cpp

```diff
@@ -1109,6 +1112,41 @@ class ProteusJitPassImpl {
       }
     }
   }
+
+  void registerLambdaFunctions(Module &M) {
+      DEBUG(Logger::logs("proteus-pass") << "registering lambda functions" << "\n");
+
+      SmallVector<Function *, 16> LambdaFunctions;
+      for (auto &F : M.getFunctionList()) {
+        // TODO: Demangle and search for the fully qualified proteus::jit_variable
+        // name.
+        if (F.getName().contains("register_lambda")) {
+          LambdaFunctions.push_back(&F);
+        }
+      }
+
+      for (auto Function : LambdaFunctions) {
```

Comment:
`auto *Function`

surprised `clang-tidy` didn't pick this up, it should be the draft status